app.ino

```diff
@@ -203,72 +203,126 @@ void blinkLED(int times)
 
 void switchLED(bool on)
 {
-    if(on) {
+    if (on)
+    {
         rgbLed.setColor(_rgb[1].red, _rgb[1].green, _rgb[1].blue);
-    }else {
+    }
+    else
+    {
         rgbLed.turnOff();
     }
 }
 
 void showMotionGyroSensor()
 {
-  acc_gyro->getXAxes(axes);
-  char buff[128];
-  sprintf(buff, "Gyroscope \r\n    x:%d   \r\n    y:%d   \r\n    z:%d  ", axes[0], axes[1], axes[2]);
-  Screen.print(buff);
+    acc_gyro->getXAxes(axes);
+    char buff[128];
+    sprintf(buff, "Gyroscope \r\n    x:%d   \r\n    y:%d   \r\n    z:%d  ", axes[0], axes[1], axes[2]);
+    Screen.print(buff);
 }
 
 void showPressureSensor()
 {
-  float pressure = 0;
-  float temperature = 0;
-  pressureSensor -> getPressure(&pressure);
-  pressureSensor -> getTemperature(&temperature);
-  char buff[128];
-  sprintf(buff, "Environment\r\nPressure: \r\n    %shPa\r\nTemp: %sC \r\n",f2s(pressure, 2), f2s(temperature, 1));
-  Screen.print(buff);
-}
-
-void update(){
-    switch(keepUpdate) {
-        case 1: showMotionGyroSensor();break;
-        case 2: showPressureSensor();break;
-        case 3: showHumidTempSensor();break;
-        // case 4: showMagneticSensor();break;
-    }
+    float pressure = 0;
+    float temperature = 0;
+    pressureSensor->getPressure(&pressure);
+    pressureSensor->getTemperature(&temperature);
+    char buff[128];
+    sprintf(buff, "Environment\r\nPressure: \r\n    %shPa\r\nTemp: %sC \r\n", f2s(pressure, 2), f2s(temperature, 1));
+    Screen.print(buff);
 }
 
 void showHumidTempSensor()
 {
-  ht_sensor->reset();
-  float temperature = 0;
-  ht_sensor->getTemperature(&temperature);
-  //convert from C to F
-  temperature = temperature*1.8 + 32;
-  float humidity = 0;
-  ht_sensor->getHumidity(&humidity);
-
-  char buff[128];
-  sprintf(buff, "Environment \r\n Temp:%sF    \r\n Humidity:%s%% \r\n          \r\n",f2s(temperature, 1), f2s(humidity, 1));
-  Screen.print(buff);
+    ht_sensor->reset();
+    float temperature = 0;
+    ht_sensor->getTemperature(&temperature);
+    //convert from C to F
+    temperature = temperature * 1.8 + 32;
+    float humidity = 0;
+    ht_sensor->getHumidity(&humidity);
+
+    char buff[128];
```

Comment:
again. And if each function need this kind of buffer, could we set one in large scope?